### PR TITLE
docs: fix Sphinx toctree doc build warning for performant_kernels.rst (#3863)

### PR DESCRIPTION
**Pull request by brittneylilly**
Fix Sphinx toctree warning for `performant_kernels.rst` by:
- added `:orphan:` directive to `docs/source/performant_kernels.rst` to suppress the Sphinx build warning: document isn't included in any toctree

Rationale:
- The `performant_kernels.rst`  file only contains a "TBA" placeholder, and Git history shows it has not been modified since its creation on April 17, 2024.

Resolves 1 doc build warning from issue #3863.

---
Remaining Sphinx doc build warnings after this fix (5 warnings and 1 error):

1. `torchao.prototype.quantization.Int8DynamicActivationInt4WeightConfig` - autosummary failed to import (module/attribute not found)
2. `torchao/quantization/qat/fake_quantize_config.py` (docstring of `IntxFakeQuantizeConfig`) - unexpected indentation error
3. `docs/source/eager_tutorials/finetuning.rst:34` - duplicate label `quantization-aware training (qat)` (also in `docs/source/workflows/qat.md`)
4. `docs/source/eager_tutorials/torchao_hf_integration.md:81` - image file not readable: `eager_tutorials/output.png`
5. `docs/source/workflows/inference.md:78` - duplicate label `nvidia h100` (also in `docs/source/workflows/training.md`)
6. `docs/source/workflows/inference.md:127` - duplicate label `nvidia b200` (duplicated within same file)

cc @jerryzh168 
cc @andrewor14

### CODE REVIEW

**Comment by brittneylilly** [comment]
pinging for maintainer review @jerryzh168 @vkuzo @andrewor14

**Comment by vkuzo** [comment]
@claude review

**Comment by vkuzo** [comment]
lg, please remove the .txt file and should be good to go!

**Comment by brittneylilly** [comment]
@vkuzo Done! Removed .txt file

**Comment by brittneylilly** [comment]
Hi @vkuzo, this PR failed two checks: `PR Label Check` and `Run 1xL4 Tests`
1. To resolve `PR Label Check` could a maintainer add the `module: not user facing` label, as I don't have permission to add labels on this repo?
2. `For Run 1xL4 Tests`, the failure is in the `test_inductor_from_config_params` method, which tests float8 GPU numerics, but this PR only modifies a .rst docs file. I checked PR #4571 (change to quantized_matmul.h) and it was merged on 7/14 while also failing `Run 1xL4 Tests` (#12946) with identical failures as my PR. So it appears that this failure is unrelated to either PR. 

Thank you

**Comment by vkuzo** [comment]
lg, thank you!

**Review by vkuzo — APPROVED** [review]
